### PR TITLE
DOCS-1330-Update 3.15.2 RN and version

### PR DESCRIPTION
**Pull request by bmckercher123**
Replaces: https://github.com/tigera/docs/pull/457/

Product Version(s):
CE 3.15.2

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.15/variables.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 const variables = {
-  releaseTitle: 'v3.15.1',
+  releaseTitle: 'v3.15.2',
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.15',
-  baseUrl: '/calico-enterprise/latest',
-  filesUrl: 'https://downloads.tigera.io/ee/v3.15.1',
+  baseUrl: '/calico-enterprise/3.15',
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.15/variables.js]: s/3.15/latest!

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/variables.js]: I misunderstood...I thought you meant to replace it, not add "latest"

File: calico-enterprise_versioned_docs/version-3.15/variables.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 const variables = {
-  releaseTitle: 'v3.15.1',
+  releaseTitle: 'v3.15.2',
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.15',
-  baseUrl: '/calico-enterprise/latest',
-  filesUrl: 'https://downloads.tigera.io/ee/v3.15.1',
+  baseUrl: '/calico-enterprise/3.15/latest',
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.15/variables.js]:
```suggestion
  baseUrl: '/calico-enterprise/latest',
```

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/variables.js]: Fixed.

**Review by rene-dekker — DISMISSED** [review]
lgtm

**Pull request merged**